lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -528,7 +556,11 @@ defmodule ExUnit.DocTest do
 
     case String.trim_leading(line) do
       "" ->
-        raise_incomplete_doctest(line_no, module)
+        [{previous_line, _} | _] = adjusted_lines
+
+        if not String.contains?(previous_line, " = ") do
```

Comment:
I think this check is too brittle. It will pass for == or a = inside a function call. But maybe making this check robust enough is quite complicated... So I want to try a couple things out. Can you please give me commit access to your fork in case I can figure something out? :)

Reply:
I've got the "allow edits from maintainers" thing on, so you should be able to push up commits to this.

I agree about this check. originally I was doing this check later on to check the AST of the expression, but it didn't fit very well with the error messages since at that point we didn't have the line number or module. I figured this was a decent starting point, but parsing the previous line to check the AST to make sure it's a match and not something else is definitely the safer way to go.

Reply:
Also note that we don't necessarily need spaces around `=`, and that these tests can be broken in multiple lines

```elixir
iex(1)> tuple = {1, 2}
{1, 2}
iex(2)> {1, 2}=tuple
{1, 2}
iex(3)> {1, 2}=
...(3)> tuple
{1, 2}
```